web/Dockerfile

```diff
@@ -1,3 +1,14 @@
-FROM python:3.6.0-onbuild
+FROM cloudcourseproject_base
 
-RUN touch /var/log/spa.log
+ENV APP_HOME /usr/src/app
+
+COPY static/ \
```

Comment:
If you copy all from web directory, we should remove volume definition, doesn't we?